python/nano/src/bigdl/nano/pytorch/lightning.py

```diff
@@ -47,19 +51,32 @@ def training_step(self, batch, batch_idx):
         x, y = batch
         y_hat = self._forward(batch)
         loss = self.loss(y_hat, y)
+        self.log("train_loss", loss, on_step=True, logger=True)
         return loss
 
     def validation_step(self, batch, batch_idx):
         x, y = batch
         y_hat = self._forward(batch)
         loss = self.loss(y_hat, y)
-        return loss
+        self.log("val_loss", loss, on_epoch=True, prog_bar=True, logger=True)
+        if self.metrics:
+            acc = {"val_acc_%d" % i: metric(y_hat, y) for i, metric in enumerate(self.metrics)}
```

Comment:
Not all metrics are called "acc", so I think it might be better to use the metric's name if there is one, or use it's class name.

Reply:
Applied your suggestions.